registration/registration_tests.py

```diff
@@ -1,8 +1,10 @@
 from .email import send_auth_mail
 from .models import Profile
+from conftest import PWD
 from conftest import pdfToText
```

Comment:
It is generally no good idea to import from `conftest.py` due to pytest magic. If it is really a test helper which is not using the pytest architecture adding a new `testing.py` with those common helpers is preferred.